test/server/cucumber_test.sh

```diff
@@ -0,0 +1 @@
+node ./node_modules/.bin/cucumber-js ./external/graknlabs_behaviour/**/*.feature -require-module ts-node/register --require './**/*.ts'
```

Comment:
Files should end with a newline - a lot of programs in Unix systems won't work well without it:
```suggestion
node ./node_modules/.bin/cucumber-js ./external/graknlabs_behaviour/**/*.feature -require-module ts-node/register --require './**/*.ts'

```